File: engine/engine-model/src/main/scala/tools/packet/spawn/MakeMonsterInvisible.scala
package tools.packet.spawn

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class MakeMonsterInvisible(private var _monsterObjectId: Int) extends PacketInput {
  def monsterObjectId: Int = _monsterObjectId

  override def opcode(): SendOpcode = SendOpcode.SPAWN_MONSTER_CONTROL
}

File: engine/engine-model/src/main/java/tools/packet/spawn/MakeMonsterInvisible.java
package tools.packet.spawn;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record MakeMonsterInvisible(Integer monsterObjectId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SPAWN_MONSTER_CONTROL;
   }
}